### ISSUE TITLE
Small p-value area

### ISSUE THREAD

**Issue by luisortizuriarte** [JStiles9552/Assignment_3_CLIM_680#3]
Your final figure shows a very small area where the difference in precipitation is significant between positive and negative NAO phases. Would this small area pass a _field significance_ test as we described it in class?

**Comment by pdirmeyer** [comment]
A few suggestions here:

1. Your NAO based composite is using all data based on sign (i.e., no "neutral" states for NAO). Composites based on NAO extremes (e.g., the extreme N months or magnitudes of NAO > some threshold greater than 0) could help.

2. I noticed that your Positive (31) and Negative (40) anomaly patterns should be equal and opposite if all data actually used, but they are not. There may be something wrong in the calculation - double check.
	• To debug, test at a point - what are the 71 numbers and do they make sense after compositing? 

3. The issue with field significance may be related to the possible bug above - it might clear up (it will definitely change).

**Comment by JStiles9552** [comment]
1. I changed the composite so that the positive NAO years account for the threshold greater than 0.25 and negative NAO years account for the threshold less than -0.25. This helped to increase the size of the p-value area, and therefore give the composite a greater significance.
2. I'm having trouble understanding this suggestion, but I think changing the composite as described above may have fixed this issue. If it is still not correct, please let me know what I need to do to fix it as I am unsure what else to do.
3. The field significance issue was resolved by changing the positive NAO and negative NAO thresholds, as the composite now shows much greater significance.

**Comment by JStiles9552** [comment]
One other thing, the Positive (31) and Negative (40) values that show up account for the total number of years in the climate index. Even after slicing the years so they match my dataset, they still show up on the figures and I do not know how they can be removed or changed. There may not be a way to change to them.

**Comment by luisortizuriarte** [comment]
@JStiles9552 you may want to also plot the neutral NAO in your composite to get the full picture  (and to help you debug your code.

**Comment by pdirmeyer** [comment]
Also, the literature on NAO seems to focus exclusively on DJF or DJFM (e.g., https://www.researchgate.net/profile/Geir-Ottersen/publication/304578504_An_overview_of_the_North_Atlantic_Oscillation/links/57c4146308aee465796bff48/An-overview-of-the-North-Atlantic-Oscillation.pdf). If you confine your analysis to only data from DJFM, you might get a more robust signal.

**Comment by JStiles9552** [comment]
I have followed your advice and confined my analysis to DJF for my composite by putting the dataset into groups by season and then creating a separate dataset that only selects the data within DJF. Also, I decided to increase the threshold of the Positive NAO to greater than 0.7 and the Negative NAO to less than -0.7 in order to focus only on the most extreme NAO events between 1990-2010. This lead to a much greater signal in my significance, as seen in the significance map.
